catkin_ws/src/arena_camera/src/arena_camera_node.cpp

```diff
@@ -1636,7 +1643,7 @@ const std::string& ArenaCameraNode::cameraFrame() const
 
 uint32_t ArenaCameraNode::getNumSubscribersRect() const
 {
-  return camera_info_manager_->isCalibrated() ? img_rect_pub_->getNumSubscribers() : 0;
+  return camera_info_manager_->isCalibrated() && img_rect_pub_ ? img_rect_pub_->getNumSubscribers() : 0;
```

Comment:
really, all the ptrs should be null checked
this code wasn't doing it before, so we're not making it worse, but probably should make it better a little bit at a time - sure there are no paths wherein the mgr is left uninit'd?